modules/core/src/main/java/org/apache/ignite/internal/util/GridDebug.java

```diff
@@ -360,8 +360,15 @@ public static long getCommittedVirtualMemorySize() {
     private static void initHotspotMBean() {
         if (hotspotMBean == null) {
             synchronized (GridDebug.class) {
-                if (hotspotMBean == null)
-                    hotspotMBean = getMBean(HOTSPOT_BEAN_NAME, HotSpotDiagnosticMXBean.class);
+                if (hotspotMBean == null) {
+                    try {
+                        hotspotMBean = getMBean(HOTSPOT_BEAN_NAME,
+                            Class.forName("com.sun.management.HotSpotDiagnosticMXBean"));
+                    }
+                    catch (ClassNotFoundException e) {
+                        throw new RuntimeException(e);
```

Comment:
Let's use IgniteException instead. I believe this is our alternative to a generic RuntimeException.